File: src/main/resources/export/cpp/SetpointCommand-constructor.cpp
#set($command = $helper.getByName($command_name, $robot))

#class($command.name)::#class($command.name)(double setpoint,     #class(${command.getProperty("Requires").getValue()})* #variable(${command.getProperty("Requires").getValue().toLowerCase()})) :
m_setpoint(setpoint), #variable(${command.getProperty("Requires").getValue().toLowerCase()})(#variable(${command.getProperty("Requires").getValue().toLowerCase()})){
SetName("#class($command.name)");
AddRequirements(#variable(${command.getProperty("Requires").getValue().toLowerCase()}));

File: src/main/resources/export/java/SetpointCommand-constructor.java
#set($command = $helper.getByName($command_name, $robot))

## parameters are hardcoded -- setpoint commands only take a single double argument
    private double m_setpoint;

    public #class($command.name)() {
        this(0);
    }

    public #class($command.name)(double setpoint) {
        super();
        this.m_setpoint = setpoint;

